invoke-ai/InvokeAI
# feat(ui): single getModelConfigs query

## What type of PR is this? (check all applicable)

- [z] Feature

## Description

Single query, with simple wrapper hooks (type-safe). Updated everywhere in frontend.

## QA Instructions, Screenshots, Recordings

Things that use models should work. All of this code is strictly typechecked, so we can be confident in this change.



## Merge Plan

This PR can be merged when approved
